docs/components/megamenu.md

```diff
@@ -5,123 +5,184 @@ section: components
 secondary_section: Navigation
 status: Released
 description: >-
-  ## Summary of [consumerfinance.gov](https://www.consumerfinance.gov) mega menu
-  design principles
+  <p style="padding-bottom:20px">The mega menu groups related links in a
+  compact, easy-to-access fashion for both desktop and mobile breakpoints. Our
+  mega menu is divided into five main sections: Consumer Tools, Practitioner
+  Resources, Data and Research, Policy and Compliance, and About Us.</p>
 
+  <hr>
 
-  1. The mega menu works everywhere.
+  <h2 style="padding-top:20px"> Design principles</h2>
```

Comment:
We probably shouldn't include inline styles in the markdown and should adjust the global styles if this padding is needed. (Same with the paragraph above.)